ci/test-mill-bootstrap-0.sh

```diff
@@ -7,12 +7,15 @@ git stash -u
 git stash -a
 
 # First build
-./mill -i "{__.publishLocal,assembly}"
-cp out/assembly.dest/mill ~/mill-1
+./mill -i "__.publishLocal" + assembly
```

Comment:
Is this Mill syntax?

Reply:
Yes

Reply:
* https://github.com/com-lihaoyi/mill/pull/1521